<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="VerifyLogin"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.006">
  <properties />
            <testcase name="verifyLoginPage" time="0.006" classname="testCase.VerifyLoginPage">
                <failure type="java.lang.IllegalArgumentException"
                  message="Cannot find elements when the XPath is null."
              >
<![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath is null.
  org.openqa.selenium.By$ByXPath.<init>(By.java:339)
  org.openqa.selenium.By.xpath(By.java:92)
  testCase.VerifyLoginPage.verifyLoginPage(VerifyLoginPage.java:18)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
  org.testng.TestRunner.privateRun(TestRunner.java:766)
  org.testng.TestRunner.run(TestRunner.java:587)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
  org.testng.TestNG.runSuites(TestNG.java:1039)
  org.testng.TestNG.run(TestNG.java:1007)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
